tests/test_dome_csc.py

```diff
@@ -14,7 +14,7 @@
 
 
 class CscTestCase(salobj.BaseCscTestCase, asynctest.TestCase):
-    def basic_make_csc(self, initial_state, config_dir, simulation_mode):
+    def basic_make_csc(self, initial_state, config_dir, simulation_mode, **kwargs):
```

Comment:
I added the **kwargs argument because PyCharm complains about

"Signature of method 'CscTestCase.basic_make_csc()' does not match signature of base method in class 'BaseCscTestCase'"

The BaseCscTestCase make_csc function indeed contains an additional **kwargs parameter.